docs/monitoring/README.md

```diff
@@ -16,8 +16,17 @@ Please join [SIG-Observability](https://github.com/llm-d/llm-d/blob/dev/SIGS.md#
 
 ### Helmfile Integration
 
-Provided Prometheus custom resources exist in the cluster, all [llm-d guides](../../guides/README.md) include the option to enable Prometheus
-PodMonitor creation for scraping vLLM metrics. With any llm-d helmfile example, update the modelservice values to enable monitoring:
+All [llm-d guides](../../guides/README.md) have monitoring enabled by default, supporting multiple monitoring stacks depending on the environment. We provide out of box monitoring configurations for scraping the [Endpoint Picker (EPP)](https://github.com/kubernetes-sigs/gateway-api-inference-extension/tree/main/docs/proposals/004-endpoint-picker-protocol) metrics, and vLLM metrics.
+
+See the vLLM Metrics and EPP Metrics sections below for how to further config or disable monitoring.
+
+If running on GKE, monitoring is also enabled by default
```

Comment:
Let's just remove this. We already state that prometheus stack needs to be installed in the prerequisites for each guide. Also the EPP and vLLM section below shows details on the config parameters to enable/disable so no need to mention here.

Also the `provider.name=gke` is inaccurate for enabling monitoring, it needs `inferenceExtension.monitoring.gke.enable=true`